src/behavior/src/A32NX_Interior_Generics.xml

```diff
@@ -417,12 +418,28 @@
             <UseTemplate Name="ASOBO_GT_Push_Button_Held">
                 <Condition Valid="LEFT_SINGLE_CODE">
                     <True>
-                        <LEFT_SINGLE_CODE>#LEFT_SINGLE_CODE#</LEFT_SINGLE_CODE>
-                        <LEFT_LEAVE_CODE>#LEFT_SINGLE_CODE#</LEFT_LEAVE_CODE>
+                        <Condition Valid="LEFT_LEAVE_CODE">
+                            <True>
+                                <LEFT_SINGLE_CODE>#LEFT_SINGLE_CODE#</LEFT_SINGLE_CODE>
+                                <LEFT_LEAVE_CODE>#LEFT_LEAVE_CODE#</LEFT_LEAVE_CODE>
+                            </True>
+                            <False>
+                                <LEFT_SINGLE_CODE>#LEFT_SINGLE_CODE#</LEFT_SINGLE_CODE>
+                                <LEFT_LEAVE_CODE>#LEFT_SINGLE_CODE#</LEFT_LEAVE_CODE>
```

Comment:
How does this not break toggling simvars ? Wouldn't `LEFT_SINGLE_CODE` run every time the button is released ?

Reply:
Check the existing code, exactly the same..